ui/widgets/nuxeo-collapsible.js

```diff
@@ -0,0 +1,155 @@
+   * `--nuxeo-collapsible-icon-height` | Chevron height | `20px`
+   * `--nuxeo-collapsible-icon-width` | Chevron width | `20px`
+   *
+   * @memberof Nuxeo
+   *
+   */
+
+  class Collapsible extends Nuxeo.Element {
+    static get template() {
+      return html`
+        <style>
+          .toggle {
+            --iron-icon-height: var(--nuxeo-collapsible-icon-height, 20px);
+            --iron-icon-width: var(--nuxeo-collapsible-icon-width, 20px);
+          }
+
+          button {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            margin: 0;
+            padding: var(--nuxeo-collapsible-heading-padding, 4px);
+            background: transparent;
+            border: none;
+            cursor: pointer;
+            text-decoration: none;
+            font-size: 1rem;
+          }
+
+          .heading {
+            width: calc(100% - var(--nuxeo-collapsible-icon-width, 20px));
+            overflow-x: hidden;
+            text-align: start;
+          }
+
+          .chevron {
+            height: var(--nuxeo-collapsible-icon-height, 20px);
+            width: var(--nuxeo-collapsible-icon-width, 20px);
+            display: block;
+          }
+
+          .start {
+            flex-direction: row-reverse;
+            justify-content: flex-end;
+          }
+
+          iron-collapse {
+            padding: 0 var(--nuxeo-collapsible-heading-padding, 4px);
+          }
+        </style>
+
+        <button aria-expanded="[[expanded]]" on-tap="_toggle" class$="[[chevronPosition]]">
+          <slot class="heading" name="heading"></slot>
+          <slot class="chevron" name="chevron">
+            <iron-icon class="toggle" icon="[[_toggleIcon(expanded)]]" toggle></iron-icon>
+          </slot>
+        </button>
+        <iron-collapse opened="[[expanded]]" no-animation="[[!animation]]">
+          <slot></slot>
+        </iron-collapse>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-collapsible';
+    }
+
+    static get properties() {
+      return {
+        /**
+         * Set animation to false to disable animations
+         */
+        animation: {
+          type: Boolean,
+          value: true,
+        },
+
+        /**
+         * Chevron icon position on start or end of the button
+         */
+        chevronPosition: {
+          type: String,
+          value: 'end',
+        },
+
+        /**
+         * Expended or collapsed by default
+         */
+        expanded: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
+        },
+      };
+    }
+
+    _toggle() {
+      this.expanded = !this.expanded;
+    }
+
+    _toggleIcon(expanded) {
+      return `hardware:keyboard-arrow-${expanded ? 'up' : 'down'}`;
+    }
+  }
+
+  customElements.define(Collapsible.is, Collapsible);
+  Nuxeo.Collapsible = Collapsible;
```

Comment:
iI don't recall but on my one of code reviewed by Nuno / Gabriel. I think there a check list and now we can avoid adding the definition of the element on Nuxeo -> `Nuxeo.Collapsible = Collapsible;`

Reply:
Makes sense, I was just copying the old polymer element definition. I will update it.